experimental/host/netlink/onewire.go

```diff
@@ -152,6 +151,11 @@ func newOneWire(masterID uint32) (*OneWire, error) {
 //
 
 const (
+	// Size of struct nlmsghdr.
+	sizeofNlMsghdr = 16
```

Comment:
I confirmed the value is the same everywhere, but I'd prefer to have a comment here, because it's not obvious, so something like:

    // Size of struct nlmsghdr, the value of syscall.SizeofNlMsghdr.

Reply:
This can be done in a follow up.